cassandra.cql

```diff
@@ -0,0 +1,25 @@
+DROP KEYSPACE IF EXISTS calendar;
+
+CREATE KEYSPACE IF NOT EXISTS calendar
+WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor': 1};
+
+
+USE calendar;
+
+CREATE TABLE calendar.availability (
+  listing_id INT,
+  min_stay_su tinyint,
+  min_stay_m tinyint,
+  min_stay_tu tinyint,
+  min_stay_w tinyint,
+  min_stay_th tinyint,
+  min_stay_f tinyint,
+  min_stay_sa tinyint,
```

Comment:
Looks good, but this shorthand notation seems a little confusing. I’m assuming the last part of the column name represents a day of the week? Could be confusing when handing off this schema.

Reply:
this shorthand was recommended to me and I didn't really question it and went with it. Maybe having each one with the same amount of characters would help in readability but currently it lends itself better to writing as it's shorter but I understand your concern about handing off to another dev in the future